### ISSUE TITLE
Modify: Set zero angles in new star file after star file or manual rotations.

### ISSUE THREAD

**Issue by axelbrunger** [brungerlab/pycresta#12]
Modify: Set zero angles in new star file after star file or manual rotations.

**Comment by mldylai** [comment]
Are the angles "rlnAngleRot", "rlnAngleTilt", and "rlnAnglePsi? Also conceptually why should we set angles to zero for manual rotation? Shouldn't we add/subtract the initial angle by the user specified angle?

**Comment by axelbrunger** [comment]
Dear Melody,

Yes, these are the angles. Conceptually, we set these to zero since these rotations have been applied to the
rotated images that the new star file is referring to. So, if we would leave the original angles, it would effectively
mean that subsequent programs would reapply these rotations again.

Best,
Axel



On Aug 16, 2023, at 3:19 AM, Melody Lai ***@***.***> wrote:



Are the angles "rlnAngleRot", "rlnAngleTilt", and "rlnAnglePsi? Also conceptually why should we set angles to zero for manual rotation? Shouldn't we add/subtract the initial angle by the user specified angle?
